recipes/discorpy/meta.yaml

```
@@ -0,0 +1,42 @@
+package:
+  name: discorpy
+  version: 1.4
```

This should be using templated variables to make it easy for our machinery to update the recipe:
```suggestion
{% set name = "discorpy" %}
{% set version = "1.4" %}

package:
  name: {{ name }}
  version: {{ version }}
```